frontend/src/models/Federation.model.ts

```diff
@@ -125,14 +134,22 @@ export class Federation {
 
   disableCoordinator = (shortAlias: string): void => {
     this.coordinators[shortAlias].disable();
+    this.updateActiveCoordinators();
     this.triggerHook('onCoordinatorUpdate');
   };
 
   enableCoordinator = (shortAlias: string): void => {
     this.coordinators[shortAlias].enable(() => {
+      this.updateActiveCoordinators();
       this.triggerHook('onCoordinatorUpdate');
     });
   };
+
+  updateActiveCoordinators = (): void => {
```

Comment:
For the same reason `updateActiveCoordinators` may be better named as`updateEnabledCoordinators()` and have `exchange.enabledCoordinators` instead of `exchange.activeCoordinators`.

Reply:
All changed! thanks!